.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
+env:
+  - TEST_SUITE=
```

Comment:
Is it worth separating things out a bit more with
```
  - TEST_SUITE=test:lint
  - TEST_SUITE=test:unit
```
instead of this line? Or is the length of time it takes for linting so negligible that it won't make much difference?

Reply:
No bc it's the e2e test times that are longest, as this suite grows it will be worth subsetting them. The unit tests will probably always finish first. But I will revaluate this after the caching branch in case this changes.

Reply:
OK.